src/nxt_application.h

```diff
@@ -49,7 +49,6 @@ typedef struct {
 typedef struct {
     char                       *home;
     nxt_conf_value_t           *path;
-    char                       *callable;
```

Comment:
This seems to also need to go into the commit that removes unused code.  Please check if there's anything else that should go there.

Reply:
I reorganized the commits.

Reply:
Thanks!